app/poros/movie_object.rb

```diff
@@ -0,0 +1,62 @@
+class MovieObject
+  attr_reader :title, :movie_id, :vote_average, :runtime, :genres, :overview, :cast, :reviews
```

Comment:
This is also not a big deal, but might be easier to read if it's in the same order as below. I had to jump around a bit to check them.